shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-postgresql/pom.xml

```diff
@@ -34,7 +34,11 @@
             <artifactId>shardingsphere-scaling-core</artifactId>
             <version>${project.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <scope>compile</scope>
```

Comment:
`compile` is the default value. Is it necessary?

Reply:
It needs during the compilation. Before it use the HikariCP-java7 in elasticjob-lite-core indirectly. 
Meanwhile root pom.xml in ShardingSphere has:
            <dependency>
                <groupId>com.zaxxer</groupId>
                <artifactId>HikariCP</artifactId>
                <version>${hikari-cp.version}</version>
                <scope>test</scope>
            </dependency>
The sub module will inherit it.  Therefore scope should be marked compile explicitly.

Reply:
Consider configuring the `dependencyManagement` in the upper pom.